lib/galaxy/config/sample/galaxy.yml.sample

```diff
@@ -901,8 +910,8 @@ galaxy:
   # prevent a class of attack called clickjacking
   # (https://www.owasp.org/index.php/Clickjacking).  If you configure a
   # proxy in front of Galaxy - please ensure this header remains intact
-  # to protect your users.  Uncomment and leave empty to not set the `X
-  # -Frame-Options` header.
+  # to protect your users.  Uncomment and leave empty to not set the
```

Comment:
Interestingly this and other differences are due to using [TextWrapper](https://docs.python.org/3/library/textwrap.html#textwrap.TextWrapper) from Python 3. I've improved this for both 2 and 3 in https://github.com/galaxyproject/galaxy/pull/9128 .